### PR TITLE
RAC-OKR: small POC to gather DSM usage informations

### PR DESCRIPTION
**Pull request by juliensnz**
**Description (for Contributor and Core Developer)**

Small tool tool to gather informations around DSM usage

Screenshot 2020-10-01 at 23 34 51

Screenshot 2020-10-12 at 10 38 56

Screenshot 2020-10-12 at 10 41 57

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Todo
| Added legacy Behats               | Todo
| Added acceptance tests            | Todo
| Added integration tests           | Todo
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

**Review by ValentinMumble — APPROVED** [review]

File: akeneo-design-system/package.json [hunk 1]
```diff
@@ -49,6 +49,7 @@
     "eslint-plugin-react": "^7.20.6",
     "eslint-plugin-react-hooks": "^4.1.0",
     "http-server": "^0.12.3",
+    "glob": "^7.1.6",
```
Comment (StevenVAIDIE) [inline akeneo-design-system/package.json:52]: Why yarn.lock doesn't changed ?

**Review by tseho — APPROVED** [review]

File: akeneo-design-system/bin/dsm [hunk 2]
```diff
@@ -83,11 +106,95 @@ const exportComponentToIndex = async (path, componentName) => {
...
+    isTest: 0,
+    isShared: 0,
+    isComponent: 0,
+    usesStyledComponents: 0,
+    usesPimShared: 0,
+    usesLegacyBridge: 0,
+    usesBEM: 0,
+    usesDSM: 0,
+    usesDSMOverStyledComponents: 100,
+    usesDSMOverBEM: 100
```
Comment (StevenVAIDIE) [inline akeneo-design-system/bin/dsm:166]: I think you can remove those lines

**Review by SamirBoulil — APPROVED** [review]

**Review by StevenVAIDIE — APPROVED** [review]

File: akeneo-design-system/bin/dsm [hunk 3]
```diff
@@ -2,26 +2,49 @@
 
 const fs = require('fs');
 const yargs = require('yargs');
+const glob = require('glob');
 
-const {ComponentName} = yargs
+const {_ : commandName, ComponentName, sourceFolder} = yargs
```
Comment (ValentinMumble) [inline akeneo-design-system/bin/dsm:7]: Just a little prettier run on this and 👌

File: akeneo-design-system/package.json (hunk 1 above)
Comment (juliensnz) [reply akeneo-design-system/package.json:52]: it seems that it was already a peer dependency

**Pull request merged**